seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/schema/SchemaChangeManager.java

```diff
@@ -0,0 +1,418 @@
+                        .append(" ")
+                        .append(quoteIdentifier(event.getColumn().getName()))
+                        .append(" ")
+                        .append(typeDefine.getColumnType());
+        if (event.getColumn().getComment() != null) {
+            sqlBuilder
+                    .append(" ")
+                    .append("COMMENT ")
+                    .append("'")
+                    .append(event.getColumn().getComment())
+                    .append("'");
+        }
+        if (event.getAfterColumn() != null) {
+            sqlBuilder.append(" ").append("AFTER ").append(quoteIdentifier(event.getAfterColumn()));
+        }
+
+        String modifyColumnSQL = sqlBuilder.toString();
+        if (!execute(modifyColumnSQL, tablePath.getDatabaseName())) {
+            log.warn("Failed to alter table modify column, SQL:" + modifyColumnSQL);
+        }
+    }
+
+    public void applySchemaChange(TablePath tablePath, AlterTableAddColumnEvent event)
+            throws IOException {
+        BasicTypeDefine typeDefine = DorisTypeConverterV2.INSTANCE.reconvert(event.getColumn());
+        StringBuilder sqlBuilder =
+                new StringBuilder()
+                        .append("ALTER TABLE")
+                        .append(" ")
+                        .append(tablePath.getFullName())
+                        .append(" ")
+                        .append("ADD COLUMN")
+                        .append(" ")
+                        .append(quoteIdentifier(event.getColumn().getName()))
+                        .append(" ")
+                        .append(typeDefine.getColumnType());
+        if (event.getColumn().getDefaultValue() != null
+                && isSupportDefaultValue(event.getColumn())) {
+            sqlBuilder
+                    .append(" DEFAULT ")
+                    .append(quoteDefaultValue(event.getColumn().getDefaultValue()));
+        }
+        if (event.getColumn().getComment() != null) {
+            sqlBuilder
+                    .append(" ")
+                    .append("COMMENT ")
+                    .append("'")
+                    .append(event.getColumn().getComment())
+                    .append("'");
+        }
+        if (event.getAfterColumn() != null) {
+            sqlBuilder.append(" ").append("AFTER ").append(quoteIdentifier(event.getAfterColumn()));
+        }
+
+        String addColumnSQL = sqlBuilder.toString();
+        if (!execute(addColumnSQL, tablePath.getDatabaseName())) {
+            log.warn("Failed to alter table add column, SQL:" + addColumnSQL);
+        }
+    }
+
+    /**
+     * Support Default Value
+     *
+     * @param column
+     * @return
+     */
+    // todo support more type
+    private boolean isSupportDefaultValue(Column column) {
+        switch (column.getDataType().getSqlType()) {
+            case STRING:
+            case BIGINT:
+            case INT:
+            case TIMESTAMP:
```

Comment:
Is it still supported `TIMESTAMP` in the case of oracle-cdc?

Reply:
here is doris sink , nothing to do with oracle

Reply:
mysql is compatible with doris ， your case source is mysql-cdc, bug source is oracle-cdc, will it still work?

Reply:
This is the default type supported by doris, if oracle unsupported ,it will not  Include  'TIMESTAMP'